Recentupdate

```diff
@@ -0,0 +1,266 @@
+<?php
+$strValidateMessage = "";
+
+
+
+// check for form submission
+if (isset($_POST['submit'])) {
+
+    // inputs from browser
+	$name     = trim($_POST['name']);
+	$phone    = trim($_POST['phone']);
+	$email    = trim($_POST['email']);
+	$suburb   = trim($_POST['suburb']);
+	$dropdown = trim($_POST['dropdown']);
+	$message  = trim($_POST['message']);
+
+
+    // email recipient as subject
+	$to      = "contact@rawadmerhi.com";
+	$subject = "New Message";
+
+    // validate inputs
+	$errors = validateInputs($name,$phone,$email,$suburb,$message);
+
+	if (empty($errors)) {
+         // no errors, send the mail
+		if (mail($to, $subject, $message, "From: $name <$email>")) {
```

Comment:
If you look at the PHP spec for this function, you will see the third parameter of it is the message body of the email. In this case, you are passing in $message which is just the message field from your form (doesn't contain any of the other info like phone, etc.). 

Let's change this to contain all the fields. I will post an example.